php-nginx/php.ini

```diff
@@ -478,7 +478,7 @@ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
 ; Development Value: On
 ; Production Value: Off
 ; http://php.net/display-errors
-display_errors = Off
+display_errors = stderr
```

Comment:
Yes, but if `error_log` is set, it writes there also. I don't think we care to write two files with the same logs. Especially if we set `log_errors` to `On`